core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -24,8 +24,14 @@ oppia.controller('Dashboard', [
       $scope, $rootScope, $window, oppiaDatetimeFormatter, alertsService,
       DashboardBackendApiService, RatingComputationService,
       ExplorationCreationService, FATAL_ERROR_CODES, UrlInterpolationService) {
+    var EXP_PUBLISH_TEXTS = {
+      default: 'This exploration is private. Publish it to receive statistics.',
+      sm: 'Publish the exploration to receive statistics.'
```

Comment:
Is this because the default message is too long on mobile? Is there a way to just have this take two lines on smaller screens, or just use the shorter message for both views?